twoliter/embedded/Makefile.toml

```diff
@@ -668,6 +668,24 @@ if [[ -d ${migrations_root} ]]; then
     done
 fi
 
+# Third pass: Check for missing version transitions in Release.toml
+
+mapfile -t transitions < <(grep -oP '\(\K[0-9]+\.[0-9]+\.[0-9]+,\s*[0-9]+\.[0-9]+\.[0-9]+(?=\))' Release.toml | sort -V)
+
+if [[ ${#transitions[@]} -gt 0 ]]; then
+    prev_to=""
+    for transition in "${transitions[@]}"; do
```

Comment:
This check would pass even if migrations stop at an old version. For example, if `Release.toml` declares `version = "1.50.0"` but the last transition is `"(1.47.0, 1.48.0)"`, the
check passes.

After the loop, consider verifying that the last transition ends at the current version:

```bash
if [[ -n "${prev_to}" && "${prev_to}" != "${version}" ]]; then
    problems+=("Last transition ends at ${prev_to}, but current version is ${version}")
fi
```